src/move_basic.cpp

```diff
@@ -457,81 +452,40 @@ void MoveBasic::executeAction(const move_base_msgs::MoveBaseGoalConstPtr& msg)
     bool reverseWithoutTurning =
         (reverseWithoutTurningThreshold > dist && linear.x() < 0.0);
 
-    // Initial rotation to face goal
-    for (int i=0; i<rotationAttempts; i++) {
```

Comment:
Am i correct to assume there is no more rotation attempts in the new move_base?

Reply:
I have implemented a proportional controller, which should be tuned so, that this does not happen. Since I was testing move_basic I couldn't reproduce this to be actually useful, because it always stopped fine. @JanezCim If you think this feature is crucial I can re-added, no problem

Reply:
No actually i dont care much about it, never needed it, just wanted to know